stable/grafana/README.md

```diff
@@ -31,7 +31,10 @@ The command removes all the Kubernetes components associated with the chart and
 
 | Parameter                                 | Description                         | Default                                           |
 |-------------------------------------------|-------------------------------------|---------------------------------------------------|
-| `server.image`                            | Container image to run              | grafana/grafana:5.0.4                             |
+| `server.image.repository`                 | image repository                    | grafana (in dockerhub)                            |
+| `server.image.name`                       | image name                          | grafana                                           |
```

Comment:
I'm not necessarily against this, but you're aware it's fairly uncommon to do this, right? In general it's agreed that `repository` should be the entire name of the image. At least in the Charts I tend to use.

Reply:
Yea, I'm fine with doing it either way. The charts I've used have followed the pattern you've experienced. That being said, at least for my use case, I've found it rather nice to have the "registry" portion in its own variable. It makes overriding that one portion the smallest possible change, with my overrides not even caring about the actual "name" of the image.